### ISSUE TITLE
Lost track of my release - how can I get it back?

### ISSUE THREAD

**Issue by tjarjoura** [helm/helm#7357]
For context I'm using helm2 with the tillerless helm plugin(https://github.com/rimusz/helm-tiller). I created a release about 2 months ago and somehow my local tiller installation got removed and re-installed. Now the release does not show up when running `helm ls`, but the resources are still up in my cluster. Is there any way for me to "import" it back into my current installation of helm so I can continue to manage these resources? I can see that the release data is still stored under the k8s secret $RELEASE_NAME.v24. 

Output of `helm version`:
```
Client: &version.Version{SemVer:"v2.16.1", GitCommit:"bbdfe5e7803a12bbdf97e94cd847859890cf4050", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.16.1", GitCommit:"bbdfe5e7803a12bbdf97e94cd847859890cf4050", GitTreeState:"clean"}
```

Output of `kubectl version`:
```
Client Version: version.Info{Major:"1", Minor:"14+", GitVersion:"v1.14.7-eks-1861c5", GitCommit:"1861c597586f84f1498a9f2151c78d8a6bf47814", GitTreeState:"clean", BuildDate:"2019-09-24T22:12:08Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"13+", GitVersion:"v1.13.12-eks-eb1860", GitCommit:"eb1860579253bb5bf83a5a03eb0330307ae26d18", GitTreeState:"clean", BuildDate:"2019-12-23T08:58:45Z", GoVersion:"go1.11.13", Compiler:"gc", Platform:"linux/amd64"}
```

Cloud Provider/Platform (AKS, GKE, Minikube etc.):  AWS

**Comment by bacongobbler** [comment]
If the configmap is still there, it should be accessible assuming tiller is correctly running. Removing and re-installing tiller is a fairly common pattern.

Is there anything in tiller's logs that would indicate why it cannot access or see those releases?

**Comment by tjarjoura** [comment]
I think I figured out the issue. Basically I was using two different k8s clusters and it seems that the local tiller just uses the one from the current context when it is started, so the releases in the other cluster were invisible. I was able to fix this by stopping tiller, switching context to the other cluster, and then starting tiller. So you're right it had nothing to do with me re-installing tiller at all.
